src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/hmi/ui_is_ready_request.h

```diff
@@ -80,9 +80,12 @@ class UIIsReadyRequest : public app_mngr::commands::RequestToHMI,
   /**
    * @brief Send request to HMI for fetching of cappabilities
    */
-  void SendMessageToHMI();
+  void RequestCapabilities();
 
  private:
+  std::vector<hmi_apis::FunctionID::eType>
+      requests_required_for_UI_capabilities_;
```

Comment:
@ydementieiev Maybe to rename all these long vars so more short names? I don't insist.
 - `requests_required_for_UI_capabilities_` -> `requests_for_UI_capabilities_`
 - `requests_required_for_TTS_capabilities_;` -> `requests_for_TTS_capabilities_;`
and so on...

Reply:
@AByzhynar  I leave old variant, it is longer but for me personally it turned out to be more understandable